hdfs_namenode/datadog_checks/hdfs_namenode/hdfs_namenode.py

```diff
@@ -92,24 +92,30 @@ class HDFSNameNode(AgentCheck):
     def check(self, instance):
         jmx_address = instance.get('hdfs_namenode_jmx_uri')
         disable_ssl_validation = instance.get('disable_ssl_validation', False)
+        tags = instance.get('tags')
         if jmx_address is None:
             raise Exception('The JMX URL must be specified in the instance configuration')
 
+
+        namenode_url = ['namenode_url:' + jmx_address]
```

Comment:
No need for a intermediate `namenode_url` variable, and this we're adding only 1 item we should use `append`:

```python
tags.append('namenode_url:' + jmx_address)
```